Docs/Migration-INI.md

```diff
@@ -0,0 +1,854 @@
+DefaultValue=false
+SettingSection=Video
+SettingKey=UseDDWrapperForMapEditor
+```
+
+2. **OPTIONAL (YR)** Add sections:
+```ini
+[GameOptionsPanelExtraControls]
+; Enable if using Phobos
+;0=chkTooltipsExtra:SettingCheckBox
+;1=chkPrioritySelection:SettingCheckBox
+;2=chkBuildingPlacement:SettingCheckBox
+
+[chkTooltipsExtra]
+Location=24,151, ;12,151
+Text=Sidebar Tooltip Descriptions
+ToolTip=Enables additional information in sidebar tooltips.
+DefaultValue=true
+ParentCheckBoxName=chkTooltips
+ParentCheckBoxRequiredValue=true
+SettingSection=Phobos
+SettingKey=ToolTipDescriptions
+
+[chkPrioritySelection]
+Location=242,54
+Text=Mass Selection Filtering
+ToolTip=If enabled, non-combat units are not selected if mass-selecting together with combat units.
+DefaultValue=false
+SettingSection=Phobos
+SettingKey=PrioritySelectionFiltering
+
+[chkBuildingPlacement]
+Location=242,78
+Text=Show Building Placement Preview
+ToolTip=If enabled, shows a preview image of the building when placing it.
+DefaultValue=false
+SettingSection=Phobos
+SettingKey=ShowBuildingPlacementPreview
+```
+
+3. Add sections:
+```ini
+[lblPlayerName]
+Location=12,195
+
+[tbPlayerName]
+Location=113,193
+
+[lblNotice]
+Location=12,220
+
+[btnConfigureHotkeys]
+Location=12,290
+
+[chkDisablePrivateMessagePopup]
+Location=12,138
+Text=Disable private message pop-ups
+
+[chkAllowGameInvitesFromFriendsOnly]
+Location=276,68
+Text=Only receive game invitations@from friends
+
+[lblAllPrivateMessagesFrom]
+Location=276,138
+
+[ddAllowPrivateMessagesFrom]
+Location=470,137
+
+[gameListPanel]
+Location=0,200
+```
+
+## Create new `PlayerExtraOptionsPanel.ini`
+```ini
+[btnClose]
+Location=220,0
+Size=18,18
+
+[lblHeader]
+Location=12,6
+
+[chkBoxForceRandomSides]
+Location=12,28
+
+[chkBoxForceRandomColors]
+Location=12,50
+
+[chkBoxForceRandomTeams]
+Location=12,72
+
+[chkBoxForceRandomStarts]
+Location=12,94
+
+[chkBoxUseTeamStartMappings]
+Location=12,130
+
+[btnHelp]
+Location=160,130
+
+[lblPreset]
+Location=12,156
+
+[ddTeamStartMappingPreset]
+Size=157,21
+Location=65,154
+
+[teamStartMappingsPanel]
+Location=12,189
+```
+
+## Appendix
+
+For completion's sake, below are additional steps required for a complete migration (beyond INIs).
+
+### Replace binary files
+
+1. Replace contents of `Resources/Binaries` and `Resources/Compatibility` with new files.
+2. Replace your launcher executable in root directory (whetever it's called, `YRLauncher.exe`, `MentalOmegaLauncher.exe`, etc.).
+3. Remove `Resources/clientdx.exe`, `Resources/clientxna.exe` and `Resources/clientogl.exe`. In their place create these scripts (replace `YRLanucher.exe` inside with your main client executable name):
+`Resources/clientdx.bat`:
+```
+cd ..
+YRLauncher.exe -DX
+```
+`Resources/clientogl.bat`:
+```
+cd ..
+YRLauncher.exe -OGL
+```
+`Resources/clientxna.bat`:
+```
+cd ..
+YRLauncher.exe -XNA
+```
+
+### Add new assets
+
+Every file here can be either in `Resources` or in theme directories:

```

Comment:
Could you leave a note about where to find these files? For example, perhaps in the mod base?